drakrun/drakrun/draksetup.py

```diff
@@ -594,27 +594,24 @@ def postinstall(report, generate_usermode):
 
     logging.info("Saving VM snapshot...")
 
-    # snapshot domain but don't destroy it, leave it in paused state
-    subprocess.check_output('xl save -p vm-0 ' + os.path.join(VOLUME_DIR, "snapshot.sav"), shell=True)
+    # Create vm-0 snapshot, and destroy it
+    # WARNING: qcow2 snapshot method is a noop. fresh images are created on the fly
+    # so we can't keep the vm-0 running
+    subprocess.check_output('xl save vm-0 ' + os.path.join(VOLUME_DIR, "snapshot.sav"), shell=True)
```

Comment:
If `VM-unit-tests` PR looks fine, then we can start using the `vm.save()` functions ig after it is merged

Reply:
Sure, but those changes aren't there yet ;)

Reply:
Oops